### PR TITLE
LRQA-73951 | master 

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-73951

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:bundle

**Comment by Sharryxue** [comment]
Just starting review.

File: modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase [hunk 1]
```diff
@@ -1297,23 +1299,23 @@ definition {
 			LexiconEntry.viewEntryName(rowEntry = "${accountRoleName}");
 		}
 
-		for (var accountRoleName : list "Buyer,Order Manager") {
+		for (var accountRoleName : list "Account Role 4") {
```
Comment (Sharryxue) [inline modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase]: No need to use loop here.

File: modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase [hunk 2]
```diff
@@ -1297,23 +1299,23 @@ definition {
...
 			LexiconEntry.viewNoEntry(rowEntry = "${accountRoleName}");
 		}
 
-		Pagination.viewResults(results = "Showing 1 to 4 of 6 entries.");
+		Pagination.viewResults(results = "Showing 1 to 4 of 5 entries.");
 
 		Click(locator1 = "Pagination#NEXT_LINK");
 
-		for (var accountRoleName : list "Buyer,Order Manager") {
+		for (var accountRoleName : list "Account Role 4") {
```
Comment (Sharryxue) [inline modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase]: same here.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Need changes, thanks.

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

**Comment by Sharryxue** [comment]
No more changes needed. Thanks for fixing the issue @melodywu

**Comment by Sharryxue** [comment]
Rebase and resend [Sharryxue/liferay-portal#352](https://github.com/Sharryxue/liferay-portal/pull/352).

**Pull request closed without merging**